### ISSUE TITLE
update for Blue switches?

### ISSUE THREAD

**Issue by smenzer** [ryanjohnsontv/node-red-contrib-ha-inovelli-manager#51]
I'm using these nodes for my Red switches, but I just bought a bunch of Blue switches that I'm using via zigbee2mqtt.  Any chance you could add nodes to support these switches, too?  Thanks!

**Comment by ryanjohnsontv** [comment]
I’ll try to get a version out tomorrow with support for the Blue switches! I don’t have any personally so it might be a bit of trial and error to have users like yourself test out the nodes and make sure all the buttons are mapped properly, but rest assured I’ll make myself available to get it working as expected!

**Comment by smenzer** [comment]
I will be more than happy to help test...just let me know when and how I can help!

**Comment by ryanjohnsontv** [comment]
I've started a pretty hefty refactor to include ZHA and Zigbee2MQTT: https://github.com/ryanjohnsontv/node-red-contrib-ha-inovelli-manager/pull/52

It's taking a bit of time to fine tune everything, but I've got the scene management done. Next up I'm going to try and get it figured out for the notification manager and release then. I wanted to see if there was an existing blueprint or script you've been using in the meantime that I could reference to know how the commands are sent? I've tried looking at public blueprints but they all seem to differ so I'm not certain which is the most accurate way.
